CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
+## Supported Releases 5.0.0 and 6.0.0
+### Summary
+
+This double release adds new Puppet 4 features: data in modules, EPP templates, the $facts hash, and data types. The 5.0.0 release is fully backwards compatible to existing Puppet 4 configurations and provides you with [deprecation warnings](https://github.com/puppetlabs/puppetlabs-stdlib#deprecation) for every argument that will not work as expected with the final 6.0.0 release. See the [stdlib docs](https://github.com/puppetlabs/puppetlabs-stdlib#validate_legacy) for an in-depth discussion of this.
```

Comment:
shouldn't this be: "the 5.0 release is fully backwards compatible to existing Puppet *3* configurations" ...?